modules/reissue-internal-certificates-secured-clusters.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+:_module-type: PROCEDURE
+[id="reissue-internal-certificates-secured-cluster_{context}"]
+= Reissuing internal certificates for Secured Clusters using init bundles
+
+Secured clusters contain the Collector, Sensor, and Admission Control components. These components use a built-in server certificate for authentication when communicating with other {product-title} components.
+
+The RHACS portal shows an information banner when the Central certificate is about to expire.
+
+[NOTE]
+====
+The information banner only appears 15 days before the certificate expiry date.
+====
+
+.Prerequisites
+
+* To reissue certificates, you must have `write` permission for the `ServiceIdentity` resource.
+
+[IMPORTANT]
+====
+Store this bundle securely because it contains secrets.
+You can use the same bundle on multiple secured clusters.
+====
+
+.Procedure
+
+To generate an init bundle using the {product-title-short} portal:
```

Comment:
Since you have two ways to do this (iiuc), probably stick a bullet on this line and on line 38 (CLI method). Then make the `.` steps into `..` steps under the `*`